spacy_transformers/layers/listener.py

```diff
@@ -36,7 +36,10 @@ def backprop_and_clear(self, *args, **kwargs):
         clears it. This saves memory, as otherwise we hold onto that callback
         until the next batch.
         """
-        result = self._backprop(*args, **kwargs)
+        if self._backprop is not None:
+            result = self._backprop(*args, **kwargs)
+        else:
+            result = None
```

Comment:
Seems reasonable :-) The only thing I worry about is that the `backprop` could be `None` by accident (e.g. some custom model not returning the backprop call correctly) and that this code then will not crash, and the user might thus not notice that the backpropagation is not actually happening fully...